doc/classes/GeometryInstance3D.xml

```diff
@@ -108,13 +113,13 @@
 			The standard texel density for lightmapping with [LightmapGI].
 		</constant>
 		<constant name="LIGHTMAP_SCALE_2X" value="1" enum="LightmapScale">
-			Multiplies texel density by 2× for lightmapping with [LightmapGI]. To ensure consistency in texel density, use this when scaling a mesh by a factor between 1.5 and 3.0.
+			Multiplies texel density by 2x for lightmapping with [LightmapGI]. To ensure consistency in texel density, use this when scaling a mesh by a factor between 1.5 and 3.0.
```

Comment:
Small change here to avoid using a Unicode character for `x` that might be unexpected.  Should help with screen readers or systems with limited font sets.

Reply:
I would prefer keeping the Unicode character. Godot bundles its own font which includes a × symbol, and screen readers should be able to read "×" as "times" by now.

Reply:
Fair enough, will revert.